pkg/interface/src/views/apps/publish/components/lib/Notebook.tsx

```diff
@@ -89,7 +90,11 @@ export function Notebook(props: NotebookProps & RouteComponentProps) {
           </Link>
         )}
         {!isOwn && (
-          <Button ml={isWriter ? 2 : 0} error border>
+          <Button ml={isWriter ? 2 : 0} error border onClick={() => {
+            api.publish.unsubscribeNotebook(deSig(ship), book).then(() => {
+              history.push("/~publish");
+            });
+          }}>
```

Comment:
nit: we should add `<Spinner>` to this to show some reactivity, a la how Chat unsubscribes work.

Reply:
![Okily dokily](https://memegenerator.net/img/instances/81331302/okily-dokily.jpg)